docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.1"
+
+services:
+  sonos:
+    tty: true
+    stdin_open: true
+    expose: 
+      - 1915
+    ports:
+      - 1915:1915
+    image: maxandersen/sonos_exporter
```

Comment:
this should of course be changed to your dockerhub based builds if you want to merge this.